src/internal/HtmlColor.h

```diff
@@ -76,6 +90,23 @@ struct HtmlColor
     };
 
     // ------------------------------------------------------------------------
+    // Parse a HTML4/CSS3 color name
+    //
+    // name - the color name
+    //
+    // It accepts all standard HTML4 names and, if USE_CSS3_COLORS macro is
+    // defined, the color names defined in CSS3 also
+    // ------------------------------------------------------------------------
+    bool Parse(const char* name);
+
+    // ------------------------------------------------------------------------
+    // Converts this color code to its HTML4/CSS3 name
+    //
+    // name - array of MAX_HTML_COLOR_NAME_LEN character to store the name
+    // ------------------------------------------------------------------------
+    void ToString(char name[MAX_HTML_COLOR_NAME_LEN]) const;
```

Comment:
In this case, we known the maximum length of the output string and we can just request a properly sized array.
We can pass also the buffer size just in case someone would like to allocate the string in the heap but doesn't like to use "new char[MAX_HTML_COLOR_NAME_LEN]". But it would add additional code to the library.